incydr/cli/core.py

```diff
@@ -40,6 +50,68 @@ def format_help(self, ctx: click.Context, formatter: click.HelpFormatter) -> Non
         )
 
 
+class ExceptionHandlingGroup(IncydrGroup):
+    """A `click.Group` subclass to add custom exception handling."""
+
+    _original_args = None
+    client = None
+
+    def __init__(self, *args, **kwargs):
+        self.client = Client()
```

Comment:
I think it would maybe be better to just grab the logger instead of the whole client here, but I'm not sure if there's a good way to do that

Reply:
You can create a new instance of `IncydrSettings` to get just the logger.

```
>>> client = incydr.Client()
>>> settings = incydr._core.settings.IncydrSettings()
>>> client.settings.logger is settings.logger
True
```

Reply:
updated to skip the auth `refresh()` call in this instance by passing `skip_auth=True` to the client init call